pontoon/base/models.py

```diff
@@ -255,6 +368,23 @@ def add_translation(self, string, locale, user):
         translation.save()
 
 
+class TranslationManager(models.Manager):
+    def get_queryset(self):
+        return (super(TranslationManager, self).get_queryset()
+                .filter(deleted__isnull=True))
+
+
+class DeletedTranslationManager(models.Manager):
+    def get_queryset(self):
+        return (super(DeletedTranslationManager, self).get_queryset()
+                .filter(deleted__isnull=False))
+
+
+def extra_default():
+    """Default value for the Translation.extra field."""
+    return {}
```

Comment:
This seems a bit silly. Why not just `extra = JSONFiled(default=lambda: {})`? Putting it out in a separate function makes me wonder what is so fancy about this function while I'm reading the model definition.